release-team/role-handbooks/docs/README.md

```diff
@@ -1,387 +1,543 @@
+1. Enable branch protection on the new `dev-` branch and deprecate the older one, e.g: https://github.com/kubernetes/test-infra/pull/11984
+    
 
-3. Send the playbook to sig-docs slack and mailinglist for review
 
-4. Remove this file: https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/docs/update-release-docs-old.md (old gh-pages doc). After merging, delete this line.
+1. Create milestone for NEW upcoming release. Depending on your permissions, you might need to contact a SIG Docs maintainer. Move anything missed for the current release to the new milestone.
 
----
+1. Update the Netlify (contact a [SIG Docs chair](https://github.com/kubernetes/community/tree/master/sig-docs#leadership) if you do not have access and they can assist with this):
 
-## Appendix I: Setup and tools
+    - Login to [Netlify](https://app.netlify.com/) and navigate to the Sites tab.
 
-You will need to use [Git source control management](https://git-scm.com/) with the [GitHub](https://github.com/) site.
+      - Clean Up
+        - Delete the oldest docs site (usually a deprecated version 5+ releases ago)
 
-Make sure you have a GitHub account and are properly setup with the following:
+      - Update [vnext](https://app.netlify.com/sites/kubernetes-io-vnext-staging/settings) staging
+        - Point at the new future dev-[future release]
+        - Taking the defaults here is mostly fine
+        - When in doubt, compare it to a working example
+        - (check) build only production branch
 
-*   Register your GitHub account with Google at [https://github.corp.google.com/](https://github.corp.google.com/)
-*   Sign up at the CNCF (Linux Foundation) as an employee of Google at [https://identity.linuxfoundation.org/projects/cncf](https://identity.linuxfoundation.org/projects/cncf), login with your Google account.
-*   Set your [primary email in GitHub](https://github.com/settings/emails) to match your Google email address used in the CNCF step.
-*   Make sure your email address in Git (either command line or client) is also set to your Google email address.
-*   Request to be added to the following GitHub organizations/teams:
-    *   Google - [https://github.com/google](https://github.com/google)
-    *   Google Cloud Platform - [https://github.com/GoogleCloudPlatform](https://github.com/GoogleCloudPlatform)
-    *   Kubernetes - [https://github.com/kubernetes](https://github.com/kubernetes)
-    *   [kubernetes.github.io-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-github-io-maintainers) team as an admin
-    *   [sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) team as a member
+      - Create a site for when the current release rolls from master to release-[version]. For example, for the 1.14 release, we needed to create a release-1.14 site that wont come into play until the future lead rolls the sites.
+        - Taking the defaults here is mostly fine
```

Comment:
Seems confusing and spelling nit 😸: Can you clarify, For example ... 
The list of `what to do` seems to be clear, though.

Reply:
You're on fire! Thanks! Instead of clarifying "For example, ..." what if the whole bullet said: "You'll need to create a Netlify site that builds from `release-[future release]` branch. Even though the `[future release]` is currently `master` (kubernetes.io), eventually `master` will be PR'd to reflect a newer k8s version and we'll use the `release-[future release]` branch to contain all prior changes - like a snapshot. (https://v1-13.docs.kubernetes.io)"

Idk, maybe it's getting worse 😆

Reply:
Just a ... spelling geek 🤓.
This seems clear:
Create a Netlify site which builds from the release-[future release] branch.

The rest is confusing. Not sure if I have this correct.
I am guessing that you are explaining the branch swapping that happens during a release: release-[future-release] represents the working master branch or the current k8s docs site. Contributors make changes to the documentation (master branch) and the changes are published immediately. During a release, though, the release-[future-dev] branch is active and eventually merged with the working master branch to become the next release. After the release, release-[future-release] represents a prior version of the documentation, and a new future release branch is created pointing to master.

Reply:
You got it! And it is confusing. What makes it more confusing is sticking with the idea of `[current-release]` / `[future-release]` as "static" descriptors. It could be interpreted that they represent the same version throughout the document, however in the context of _anything_ after release day should mean `current-release` = `future-release` and `future-release` = `future-release+1`

To avoid this rabbit hole, I tried to remain consistent with them (no interpretation) throughout the release.

I'm open to alternative suggestions, otherwise I'll leave it in the modified format for now.

Reply:
+1
In the future, a `diagram` may be a good fit for explaining the docs release cycle and the branching process for docs development.

Reply:
+ https://github.com/kubernetes/website/pull/14139 for a reminder nit